### PR TITLE
Add MYPYPATH To Stubtest Documentation

### PR DESCRIPTION
**Pull request by HassanAbouelela**
MYPYPATH must be set to the stub directory when running subtest on third party packages, otherwise the stubs won't be found. This is already done in scripts, but not mentioned in the relevant documentation.

### CODE REVIEW

File: tests/README.md [hunk 1]
```diff
@@ -116,7 +116,7 @@ at [mypy](https://github.com/python/mypy/issues).
 To run stubtest against third party stubs, it's easiest to use stubtest
```
Comment (hauntsaninja) [inline tests/README.md:116]: Want to move this paragraph into the stubtest_third_party section where you made the other change? (I think this text was written before we tested third party packages with stubtest, which is how this text ended up in the stdlib section)

**Review by hauntsaninja — COMMENTED** [review]
Thanks for updating this, one comment

**Comment by AlexWaygood** [comment]
This PR conflicts a little with #8822. Not sure how we want to resolve that.

File: tests/README.md (hunk 1 above)
Comment (HassanAbouelela) [reply tests/README.md:116]: That sounds reasonable, I'll see about doing that pending figuring out the solution with the other PR.

**Comment by HassanAbouelela** [comment]
Ah I should've done a better job looking at existing PRs. If this section is removed anyway, I'm happy to just close this PR.

I do hope some mention of it is kept, as it is still convenient to run stubtest with other options (such as concise), which isn't as easy with the script. However, toggling the ignore in metadata is something I wasn't aware of and is a good substitute.

**Comment by AlexWaygood** [comment]
> I do hope some mention of it is kept, as it is still convenient to run stubtest with other options (such as concise), which isn't as easy with the script.

That's very fair! (I'm also very happy for this to be merged before mine btw).

Maybe we should meek the example (with your revisions) and move it down to the section on `stubtest_third_party.py`? I feel like it doesn't make much sense in the current section, but your rationale for keeping it somewhere makes a lot of sense.

**Comment by hauntsaninja** [comment]
Let's merge #8822 first and once it's merged, move the example down like Alex suggests :-)

**Comment by hauntsaninja** [comment]
Okay @HassanAbouelela , just merged the other PR!

**Comment by HassanAbouelela** [comment]
Thanks, I've merged in the other changes now

File: tests/README.md (hunk 1 above)
Comment (HassanAbouelela) [reply tests/README.md]: Resolved in 6d2bd8e

File: tests/README.md [hunk 2]
```diff
@@ -136,6 +136,14 @@ check on the command line:
 (.venv3)$ python3 tests/stubtest_third_party.py Pillow toml  # check stubs/Pillow and stubs/toml
 ```
 
+To run stubtest against third party stubs, it's easiest to use stubtest
```
Comment (AlexWaygood) [inline tests/README.md:139]:
```suggestion
If you have the runtime package installed in your local virtual environment, you can also run stubtest
```

**Review by hauntsaninja — APPROVED** [review]
Thanks again, looks great!

**Pull request merged**